src/Symfony/Component/DependencyInjection/Argument/TaggedIteratorArgument.php

```diff
@@ -32,8 +33,9 @@ class TaggedIteratorArgument extends IteratorArgument
      * @param bool        $needsIndexes          Whether indexes are required and should be generated when computing the map
      * @param string|null $defaultPriorityMethod The static method that should be called to get each service's priority when their tag doesn't define the "priority" attribute
      * @param array       $exclude               Services to exclude from the iterator
+     * @param array       $excludeSelf           Whether to automatically exclude the referencing service from the iterator
```

Comment:
bool

Reply:
fixed, param added to `#[TaggedIterator]` and `#[ServiceLocator]` also. thanks